libafl_bolts/src/lib.rs

```diff
@@ -322,6 +322,8 @@ pub enum Error {
     OsError(io::Error, String, ErrorBacktrace),
     /// Something else happened
     Unknown(String, ErrorBacktrace),
+    // Error with the corpora
+    CorpusError(String, ErrorBacktrace),
```

Comment:
I would just name this `Corpus` or `CorpusInvalid` - Error is redundant.
Ideally we would map the error to one of the other errors but I'm not sure if one of them applies (maybe `IllegalState`?)

Reply:
IMO `IllegalState` seems more appropriate for when an invalid action is performed by the user or if external sources modified the items the state depends on (eg. deleting the corpus entry while the fuzzer is running leading to inaccessible corpus entry). Though we could keep it `IllegalState` for simplicity?

Reply:
Yeah, makes more sense.